cookbook/request/load_balancer_reverse_proxy.rst

```diff
@@ -83,6 +83,9 @@ In this case, you'll need to - *very carefully* - trust *all* proxies.
        $response = $kernel->handle($request);
        // ...
 
+#. Ensure that the trusted_proxies setting in your ``app/config/config.yml`` is not set or 
+   it will overwrite the ``setTrustedProxies`` call above.
```

Comment:
Just one last comment: Can you please wrap lines after the first word that crosses the 72nd character?